infra/vsts_agent_ubuntu_20_04_startup.sh

```diff
@@ -21,7 +21,7 @@ apt-get update -q
 apt-get install -qy \
   curl sudo \
   bzip2 rsync \
-  jq liblttng-ust0 libcurl3 libkrb5-3 libicu55 zlib1g \
+  jq liblttng-ust0 libcurl4 libkrb5-3 zlib1g \
```

Comment:
Do you know what we needed icu for before and why we don’t need it anymore? I know GHC uses it but we should get that from nix.

Reply:
I do not.

Reply:
I was not very satisfied with my answer here so I dug a little bit more. Unfortunately that list of dependencies has been added in one chunk with no explanation and nobody asking any question about it, back in #230. Googling for that specific list of dependencies seem to bring up mostly DotNetCore-related hits, so my best guess at this point is that these were the dependencies needed to get the Azure agent working. (It's written in C#.)

Note that I've removed the `libicu55` one because it was causing a problem (not found anymore), i.e. I have not double-checked that the other ones are still necessary.